verify/go.mod

```diff
@@ -7,7 +7,7 @@ replace sigs.k8s.io/kubebuilder-release-tools/notes => ../notes
 require (
 	github.com/google/go-github/v32 v32.1.0
 	golang.org/x/oauth2 v0.8.0
-	sigs.k8s.io/kubebuilder-release-tools/notes v0.4.0
+	sigs.k8s.io/kubebuilder-release-tools/notes v0.4.2
```

Comment:
while we can't bump this to non-existent version as we are doing in other file with image version, would not it be problem (not having our fix included in the tool) when using it from CAPI?

Reply:
This version is not used at all, see the replace in l.5.

Maybe we should just set it to v0.0.0 (if possible) to avoid confusion

Reply:
@sbueringer I see, have not expanded the file fully to see the replace.. Then it should be all good 👍🏼

Reply:
Yup. It's just pretty confusing since we changed this version to something looking almost like it makes sense a few PRs ago... :D